tests/gql_behave/tests/memgraph_V1/features/memgraph_wshortest.feature

```diff
@@ -155,3 +155,79 @@ Feature: Weighted Shortest Path
           MATCH (n {a:'0'})-[le *wShortest 10 (e, n | e.w ) w]->(m) RETURN m.a, size(le) as s, w
           """
       Then an error should be raised
+
+    Scenario: Test match wShortest with accumulated path filtered by order of ids
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:label1 {id: 1})-[:type1 {id:1}]->(:label2 {id: 2})-[:type1 {id: 2}]->(:label3 {id: 3})-[:type1 {id: 3}]->(:label4 {id: 4});
+          """
+      When executing query:
+          """
+          MATCH pth=(:label1)-[*WSHORTEST (r, n | r.id) total_weight (e,n,p | e.id > 0 and (nodes(p)[-1]).id > (nodes(p)[-2]).id)]->(:label4) RETURN pth, total_weight;
```

Comment:
does in this example the "treat nulls as zero" come into play? can you try a reverse example? with all the ids greater than the other?

Reply:
No, this is other case :) I explained about `treat nulls as zero` below in other comment